lang/sql/sql.talon

```diff
@@ -0,0 +1,38 @@
+mode: command
+and mode: user.sql
+mode: command
+and mode: user.auto_lang
+and code.language: sql
+-
+tag(): user.code_generic
+tag(): user.code_operators
+
+# Some commands have spaces around them to avoid having to continually say 'space'.
+# The ones that don't have an initial space are expected to be at the start of a new line.
+# This could probably be done in a better way.
+select: "SELECT "
+star: "* "
+distinct: "DISTINCT "
+from: "FROM "
+case: "CASE"
+when: "WHEN "
+then: "THEN "
+else: "ELSE "
+end: "END"
+as: " AS "
+where: "WHERE "
+between: " BETWEEN "
+and: "AND "
+or: "OR "
+is: "IS "
```

Comment:
I'm a little hesitant on some of the short commands (as/is/or/not/in/else/end), which is generally why other languages use e.g. `state else`

Reply:
I considered that option, but SQL (at least when dealing with relatively simple queries) tends to sound a lot like plain English. Having to say `state` before each command might be cumbersome.

Reply:
Makes sense, I can see both sides here. I'd like to run this by some other collaborators since I don't do SQL. @splondike @rntz @pokey

Reply:
`IS` is not its own operator in SQL. I do this by defining separate `IS NULL` and `IS NOT NULL` commands (https://github.com/nriley/knausj_talon/blob/nriley/lang/sql/sql.talon#L20) which is what I would recommend you do as well.

For `AND` and `OR`, these are operators so should be defined as such just like any other programming language.